event_routing_backends/backends/events_router.py

```diff
@@ -159,19 +161,21 @@ def send(self, event):
 
         for events_for_route in event_routes.values():
             for event_name, updated_event, host, is_business_critical in events_for_route:
+                func = dispatch_event_persistent if is_business_critical else dispatch_event
+                func = func if self.sync else func.delay
```

Comment:
I think the sync argument really confuses the functionality here and requires some knowledge to understand what it does. I think there should be a new backend for the bus, and it should be chosen in configuration.

Reply:
What about subclssing this with `EventBusRouter` making sure it can override those functions to process it.

Or do you want to create a new separate thing for this

Reply:
Since it was designed to have different backends, I think maybe both of these could inherit from a base class.

Reply:
This was already refactored